CliFx/CliApplication.cs

```diff
@@ -43,7 +41,7 @@ public class CliApplication : ICliApplication
             _helpTextRenderer = helpTextRenderer;
         }
 
-        private async ValueTask<int?> HandleDebugDirectiveAsync(CommandInput commandInput)
+        private async Task<int?> HandleDebugDirectiveAsync(CommandInput commandInput)
```

Comment:
Can you please change it back to ValueTask<int?>?

Reply:
Woops, must have been a merge error.